Iterable-iOS-SDK/IterableAPI.m

```diff
@@ -861,6 +861,25 @@ - (void)trackPurchase:(NSNumber *)total items:(NSArray<CommerceItem *> *)items d
 }
 
 // documented in IterableAPI.h
+- (void)updateSubscriptions:(NSArray *)emailListIds unsubscribedChannelIds:(NSArray *)unsubscribedChannelIds unsubscribedMessageTypeIds:(NSArray *)unsubscribedMessageTypeIds
+{
+    NSMutableDictionary *mutableDictionary = [[NSMutableDictionary alloc] init];
+    
+    if (_email != nil) {
```

Comment:
we should refactor this to some `addCommonFieldsToDictionary:(NSMutableDictionary dictionary)` method

Reply:
Added in addEmailOrUserIdToDictionary to make it consistent with the Android helper function.